colcon_bundle/verb/__init__.py

```diff
@@ -53,3 +53,36 @@ def check_and_mark_bundle_tool(bundle_base, *, this_build_tool='colcon'):
         os.makedirs(bundle_base, exist_ok=True)
 
     marker_path.write_text(this_build_tool + '\n')
+
+
+def check_and_mark_bundle_version(bundle_base, *, this_bundle_version):
+    """
+    Check the bundle version marker file for the previous bundle version.
+
+    The marker filename is `.bundle_version`.
+
+    :param str bundle_base: The bundle directory
+    :param str this_bundle_version: The version of the bundle previously built
```

Comment:
It seems like this is the version of the bundle that is currently being built, not previously built. Is that correct?